atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/expectations/ExpectationsAwareTransaction.java

```diff
@@ -22,7 +22,6 @@
  * Implementors of this interface provide methods useful for tracking transactional expectations and whether
  * they were breached as well as relevant metrics and alerts. Transactional expectations represent transaction-level
  * limits and rules for proper usage of AtlasDB transactions (e.g. reading too much data overall).
- * Todo(aalouane): move this out of API once part 4 is merged
  */
```

Comment:
tried to move this to client for 8 minutes but broke some imports that i couldn't fix quickly, we can look at this together

Reply:
paired offline, resolved!